msmbuilder/featurizer/featurizer.py

```diff
@@ -200,6 +200,61 @@ def partial_transform(self, traj):
         return x
 
 
+
+class StrucRMSDFeaturizer(Featurizer):
+    """Featurizer based on overall RMSD to a set of reference structures.
+
+    Parameters
+    ----------
+    atom_indices : np.ndarray, shape=(n_atoms,), dtype=int
+        The indices of the atoms to superpose and compute the distances with
+    reference_traj : md.Trajectory
+        The reference conformation to superpose each frame with respect to
+        (only the first frame in reference_traj is used)
+    superpose_atom_indices : np.ndarray, shape=(n_atoms,), dtype=int
```

Comment:
@rmcgibbo so what about the atom_indices (first parameter) above? I guess I could just rewrite all of this since I copied it over from superposefeaturizer but didn't really cross check. The description part has been updated though.